examples/taptree_of_horror/taptree_of_horror.rs

```diff
@@ -0,0 +1,314 @@
+                            and(
+                                thresh(4, pk({d2}), pk({j2}), ripemd160({g}), after({oct_24_evening})),
+                                or(pk({s2}), pk({x2}))
+                            )
+                        )
+                    ),
+                    or(
+                        or(
+                            or(
+                                and(
+                                    thresh(6, pk({a3}), pk({b3}), pk({c3}), pk({e3}), pk({f3}), pk({h3}), pk({i3}), pk({l3})),
+                                    thresh(4, pk({d3}), pk({j3}),  sha256({k}), after({oct_25_afternoon}))
+                                ),
+                                thresh(14, pk({a8}), pk({b8}), pk({c8}), pk({d8}), pk({e8}), pk({f8}), pk({h8}), pk({i8}), pk({j8}), pk({l8}), pk({s8}), pk({x8}), sha256({k}), ripemd160({g}))
+                            ),
+                            or(
+                                and(
+                                    thresh(9, pk({a4}), pk({b4}), pk({c4}), pk({d4}), pk({e4}), pk({f4}), pk({h4}), pk({i4}),  pk({j4}), pk({l4}), pk({s4}), pk({x4})),
+                                    thresh(2, sha256({k}), after({oct_26_morning}))
+                                ),
+                                and(
+                                    thresh(10, pk({a5}), pk({b5}), pk({c5}), pk({d5}), pk({e5}), pk({f5}), pk({h5}), pk({i5}),  pk({j5}), pk({l5}), pk({s5}), pk({x5})),
+                                    after({oct_26_morning})
+                                )
+                            )
+                        ),
+                        or(
+                            and(
+                                thresh(4, pk({a6}), pk({b6}), pk({c6}), pk({e6}), pk({f6}), pk({h6}), pk({i6}), pk({l6})),
+                                thresh(5, pk({d6}), pk({x6}), pk({j6}), ripemd160({g}), after({oct_25_afternoon}))
+                            ),
+                            and(
+                                thresh(4, pk({a7}), pk({b7}), pk({c7}), pk({e7}), pk({f7}), pk({h7}), pk({i7}), pk({l7})),
+                                thresh(5, pk({d7}), pk({s7}), pk({j7}), ripemd160({g}), after({oct_25_afternoon}))
+                            )
+                        )
+                    )
+                )
+            )
+
+        )"
+    )
+    .replace(&[' ', '\n', '\t'][..], "");
+
+    // make sure policy doesn't have any issues
+    let pol = Concrete::<DescriptorPublicKey>::from_str(&pol_str).unwrap();
+    let policy_desc: Descriptor<DescriptorPublicKey> = pol.compile_tr(None).unwrap();
+
+    // Now, using this public descriptor create the script address
+    let derived_descriptor = policy_desc.at_derivation_index(0).unwrap();
+    let _script_address = derived_descriptor.address(Network::Regtest).unwrap();
+    // println!("the receiving address of this script is: {}", _script_address);
+    // println!("\ndescriptor is: {}\n", desc);
```

Comment:
In 1282d30653a3cb4e188e59d5a3dd8de909aa9477:

I think you should uncomment these. It's nice to see some output from examples.